app/src/components/map/MapContainer.tsx

```diff
@@ -1053,6 +1053,7 @@ const initializeMap = (
           size_ha={sizeHa}
           state_code={stateCode}
           thumbnail={thumbnail}
+          maskDisclaimer={true}
```

Comment:
Only added for Project? Should add `true` to Plan as well `markerPlans.points`

Reply:
Good catch.

![Screenshot from 2024-10-03 14-56-04](https://github.com/user-attachments/assets/8c71fb2f-fa43-45e5-bafe-9bd00ae02b79)